tools/ci_build/amd_hipify.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+contrib_ops_path = 'onnxruntime/contrib_ops'
+core_ops_path = 'onnxruntime/core/providers'
+training_ops_path = 'orttraining/orttraining/training_ops'
+
+contrib_ops_files = [
```

Comment:
Instead of listing each file to convert, can we list all the files *not* to convert?

If someone adds a new CUDA file, it will automatically be added to the CUDA build (by cmake) but would break the AMD build until they also add the new file here.

Reply:
good idea. Will change it.